server/plugin/api.go

```diff
@@ -548,19 +548,21 @@ func (p *MatterpollPlugin) handleAddOptionConfirm(vars map[string]string, reques
 	}
 
 	prev := poll.Copy()
-	userLocalizer := p.getUserLocalizer(poll.Creator)
+	userLocalizer := p.bundle.GetUserLocalizer(poll.Creator)
 
 	if errMsg := poll.AddAnswerOption(answerOption); errMsg != nil {
 		response := &model.SubmitDialogResponse{
 			Errors: map[string]string{
-				addOptionKey: p.LocalizeErrorMessage(userLocalizer, errMsg),
+				addOptionKey: p.bundle.LocalizeWithConfig(userLocalizer, &i18n.LocalizeConfig{
+					DefaultMessage: errMsg.Message,
+					TemplateData:   errMsg.Data,
+				}),
```

Comment:
Maybe this should be wrapped into helper method?

Reply:
Good catching, thanks @hanzei 👍